demos/instance_segmentation_demo/python/instance_segmentation_demo.py

```diff
@@ -98,24 +97,30 @@ def build_argparser():
 
 
 def main():
-    log.basicConfig(format='[ %(levelname)s ] %(message)s', level=log.INFO, stream=sys.stdout)
     args = build_argparser().parse_args()
 
+    cap = open_images_capture(args.input, args.loop)
+
+    with open(args.labels, 'rt') as labels_file:
+        class_labels = labels_file.read().splitlines()
+        assert len(class_labels), 'The file with class labels is empty'
+
     # Plugin initialization for specified device and load extensions library if specified.
-    log.info('Creating Inference Engine...')
+    log.info('OpenVINO Inference Engine')
+    log.info('\tbuild: {}'.format(get_version()))
     ie = IECore()
     if args.cpu_extension and 'CPU' in args.device:
         ie.add_extension(args.cpu_extension, 'CPU')
+
     # Read IR
-    log.info('Loading network')
+    log.info('Reading model {}'.format(args.model))
     net = ie.read_network(args.model, args.model.with_suffix('.bin'))
     image_input, image_info_input, (n, c, h, w), model_type, postprocessor = check_model(net)
     args.no_keep_aspect_ratio = model_type == 'yolact' or args.no_keep_aspect_ratio
 
-    log.info('Loading IR to the plugin...')
     exec_net = ie.load_network(network=net, device_name=args.device, num_requests=2)
+    log.info('The model {} is loaded to {}'.format(args.model, args.device))
```

Comment:
Please, add `log_runtime_settings() `function in this demo

Reply:
And, I think, you should check, have you added ths in all demos or not. Because it's abscent in 3d_segmentation_demo as well.

Reply:
The function is called only in demos with AsyncPipeline, where an user can set number of inference requests/streams/threads manually. For other demos with synchronous inference the logged info is quite redundant.